topics-iam.tf

```diff
@@ -0,0 +1,96 @@
+locals {
+  read_topic_pairs = flatten([
+    for service in var.services : [
+      for topic in service.readTopics : {
+        topic = topic
+        name  = service.name
+      }
+    ]
+  ])
+  write_topic_pairs = flatten([
+    for service in var.services : [
+      for topic in service.writeTopics : {
+        topic = topic
+        name  = service.name
+      }
+    ]
+  ])
+  services = {
```

Comment:
can we rename this local var to something more meaningful?
ex: services_acls, iam_services